modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts

```diff
@@ -76,7 +76,16 @@ export default class PCFormFieldSizeController<T> implements IInputErrorNotifier
         if (!this.min) this.min = 0;
         if (!this.sizesMenu) this.setDefaultSizeType();
         this.$element.addClass('form-field');
-        this.ngModel.$render = () => this.assignValue(this.ngModel.$viewValue);
+        this.ngModel.$render = () => {
+            const rawValue = this.ngModel.$viewValue;
+
+            if (rawValue) {
+                this._sizeScale = _.findLast(this.sizesMenu,
```

Comment:
Either import lodash or use native Array methods.
```suggestion
                this._sizeScale = [...this.sizesMenu].reverse().find(
```

Reply:
Lodash has been imported.